plat/kvm/efi.c

```diff
@@ -356,8 +382,10 @@ static void uk_efi_setup_bootinfo_mrds(struct ukplat_bootinfo *bi)
 	if (unlikely(status != UK_EFI_SUCCESS))
 		UK_EFI_CRASH("Failed to free rt_mrds\n");
 
-	/* Get memory map through GetMemoryMap */
-	uk_efi_get_mmap(&map_start, &map_sz, &desc_sz);
+	/* Get memory map through GetMemoryMap and also exit Boot service.
+	 * NOTE: after exiting, EFI printing provided by BS is not available
+	 * anymore, so UK_CRASH should be used instead. */
```

Comment:
Unless it is a single line comment, we should try to not have trailing `*/` on the same line as the last line of the comment. Instead let's have it on a separate line like you did [here](https://github.com/unikraft/unikraft/pull/1105/commits/d4c51d8f0cf4b9d3a9950a7a4e5924757021cdcd#diff-4aa842463d722246e6374f63ef4cee5b61481d652a2369123dd9483383332daeR275). :pray:

Reply:
ACK